packages/core/versioning/core-version-http-server/README.md

```diff
@@ -0,0 +1,13 @@
+# @kbn/core-version-http-server
```

Comment:
Naming things, naming things... What to name things? Happy to change this. I chose to put this in a package outside `core/http` and follow the naming convention of `-server`. My guess is we will have the same thing later, but `-browser`.